From 51dec1df8d0d43e79849abac2fcbff2934538c4c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rn=20Kottmann?= <joern@apache.org>
Date: Thu, 3 Sep 2015 07:30:24 +0000
Subject: [PATCH] OPENNLP-810  POSTagger incorrectly tries to set pos
 probability to pos feature and breaks because of incompatible types. Changed
 to probabilityFeature. Thanks to Donatas Remeika for providing a patch.

git-svn-id: https://svn.apache.org/repos/asf/opennlp/trunk@1700940 13f79535-47bb-0310-9956-ffa450edef68
---
 opennlp-uima/src/main/java/opennlp/uima/postag/POSTagger.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opennlp-uima/src/main/java/opennlp/uima/postag/POSTagger.java b/opennlp-uima/src/main/java/opennlp/uima/postag/POSTagger.java
index e7d1423a1..5e77e9ddd 100644
--- a/opennlp-uima/src/main/java/opennlp/uima/postag/POSTagger.java
+++ b/opennlp-uima/src/main/java/opennlp/uima/postag/POSTagger.java
@@ -193,7 +193,7 @@ public void process(CAS tcas) {
         tokenAnnotation.setStringValue(this.posFeature, posTag);
 
         if (posProbabilities != null) {
-          tokenAnnotation.setDoubleValue(this.posFeature, posProbabilities[index]);
+          tokenAnnotation.setDoubleValue(this.probabilityFeature, posProbabilities[index]);
         }
 
         index++;
